src/example_multinomial_logistic_regression.jl

```diff
@@ -85,12 +85,12 @@ _median(β_posterior)
 
 # Quantiles
 qs = [0.05, 0.25, 0.5, 0.75, 0.95]
-quantile([β_posterior[i][1, 1] for i in 1:length(β_posterior)], qs)
-quantile([β_posterior[i][1, 2] for i in 1:length(β_posterior)], qs)
-quantile([β_posterior[i][1, 3] for i in 1:length(β_posterior)], qs)
-quantile([β_posterior[i][2, 1] for i in 1:length(β_posterior)], qs)
-quantile([β_posterior[i][2, 2] for i in 1:length(β_posterior)], qs)
-quantile([β_posterior[i][2, 3] for i in 1:length(β_posterior)], qs)
+@show quantile([β_posterior[i][1, 1] for i in 1:length(β_posterior)], qs);
```

Comment:
Thanks for noticing this. Can you do this with a loop, eg over `eachindex(first(β_posterior))`? I realize that the original code was not elegant either, but this would be a great opportunity to improve it.

Reply:
For sure.

Could you explain the syntax to me a little bit?